docs/rules/README.md

```diff
@@ -48,6 +48,7 @@ These rules enforce best practices and naming conventions:
 
 |Rule|Description|Enabled by default|
 | --- | --- | --- |
+|[aws_acm_certificate_lifecycle](aws_acm_certificate_lifecycle.md)|Disallow adding `aws_acm_certificate` resource without setting `create_before_destroy = true` in `lifecycle` block |✔|
```

Comment:
Sorry for the lack of explanation. Please fix `docs/rules/README.md.tmpl` instead, and run `go generate ./...`. The build looks failed.

https://github.com/terraform-linters/tflint-ruleset-aws/blob/c34979e4c88c750ff97b302ce56209bdc70f0b46/docs/rules/README.md.tmpl
https://github.com/terraform-linters/tflint-ruleset-aws/runs/4384564812?check_suite_focus=true